### ISSUE TITLE
Ore Generation Ideas

### ISSUE THREAD

**Issue by PierKnight** [Davoleo/Metallurgy-4-Reforged#94]
Here is a list about specific ore gen ideas:

I will update this list with people ideas ,the accepted ideas will be added in the checklist below:

  CheckList:
 - [ ] every ore needs to have his height spawn (@Luke616 idea) 
 - [x] Deep Iron spawns only in acquatic biomes
 - [x] **Platinum**:Make platinum a bit more abundant in warm zones like jungle or savanna.(@Klazkin  idea)
 - [x]  **Vulcanite**: It spawns only at lava sea level
 - [x] **Eximite**: It Spawns only on the small islands (@Luke616 idea)  )
- [x]  **shadow iron** spawns near the bedrock)  (@Luke616 idea)
- [x] Prometheum spawns only in jungle biomes
- ~~Astral Silver: spawns more frequently on mountains~~
- [x] Osmium: spawns near bedrock in overworld
- [x] sanguinite: more abundant  near fortresses
- [ ] Cerculase: cold and ice themed world gen

Ideas:
- more abundant in ice biomes.
- spaws only in plains

the point of this list is to add more variety in the world generation,this will make the ore exploration more satisfying and unique,feel free to write ideas in the comment section

**Comment by Klazkin** [comment]
**TL;DR Make platinum a bit more abundant in warm zones like jungle or savanna.** 
No realistic reason asides from the fact that more than 80% of platinum in the real world is mined in South Africa.

Much more important is that platinum is stupid rare in modded minecraft but used very often in recipes. Making it spawn in warm areas with increased chance could make platinum mining actually worthwhile, instead of being literally impossible. 

Of course, values don't have to be too significantly different. :PPPP

**Comment by PierKnight100** [comment]
Your idea has bee added in the list @Klazkin

**Comment by Davoleo** [comment]
I think there still are some nice ideas in this issue @HurtsAsHell
